src/registrar/views/application.py

```diff
@@ -0,0 +1,421 @@
+        button = request.POST.get("submit_button", None)
+        if button == "save":
+            return self.goto(self.steps.current)
+        # otherwise, proceed as normal
+        return self.goto_next_step()
+
+    def save(self, forms: list):
+        """
+        Unpack the form responses onto the model object properties.
+
+        Saves the application to the database.
+        """
+        for form in forms:
+            if form is not None and hasattr(form, "to_database"):
+                form.to_database(self.application)
+
+
+class OrganizationType(ApplicationWizard):
+    template_name = "application_org_type.html"
+    forms = [forms.OrganizationTypeForm]
+
+
+class OrganizationFederal(ApplicationWizard):
+    template_name = "application_org_federal.html"
+    forms = [forms.OrganizationFederalForm]
+
+
+class OrganizationElection(ApplicationWizard):
+    template_name = "application_org_election.html"
+    forms = [forms.OrganizationElectionForm]
+
+
+class OrganizationContact(ApplicationWizard):
+    template_name = "application_org_contact.html"
+    forms = [forms.OrganizationContactForm]
+
+    def get_context_data(self):
+        context = super().get_context_data()
+        context["is_federal"] = self.application.is_federal()
+        return context
+
+
+class AuthorizingOfficial(ApplicationWizard):
+    template_name = "application_authorizing_official.html"
+    forms = [forms.AuthorizingOfficialForm]
+
+
+class CurrentSites(ApplicationWizard):
+    template_name = "application_current_sites.html"
+    forms = [forms.CurrentSitesForm]
+
+
+class DotgovDomain(ApplicationWizard):
+    template_name = "application_dotgov_domain.html"
+    forms = [forms.DotGovDomainForm]
+
+
+class Purpose(ApplicationWizard):
+    template_name = "application_purpose.html"
+    forms = [forms.PurposeForm]
+
+
+class YourContact(ApplicationWizard):
+    template_name = "application_your_contact.html"
+    forms = [forms.YourContactForm]
+
+
+class OtherContacts(ApplicationWizard):
+    template_name = "application_other_contacts.html"
+    forms = [forms.OtherContactsForm]
+
+
+class SecurityEmail(ApplicationWizard):
+    template_name = "application_security_email.html"
+    forms = [forms.SecurityEmailForm]
+
+
+class AnythingElse(ApplicationWizard):
+    template_name = "application_anything_else.html"
+    forms = [forms.AnythingElseForm]
+
+
+class Requirements(ApplicationWizard):
+    template_name = "application_requirements.html"
+    forms = [forms.RequirementsForm]
+
+
+class Review(ApplicationWizard):
+    template_name = "application_review.html"
+    forms = []  # type: ignore
+
+    def get_context_data(self):
+        context = super().get_context_data()
+        context["Step"] = Step.__members__
+        context["application"] = self.application
+        return context
+
+    def goto_next_step(self):
+        return self.done()
+        # TODO: validate before saving, show errors
+        # forms = self.get_all_forms()
```

Comment:
**suggestion (non-blocking)**: Say more about why this is a TODO, why the code commented out doesn't just work. Maybe link to a ticket that explains more?

Reply:
Done.